### PR TITLE
making bot less angry

### PR DESCRIPTION
**Pull request by KarolinaGroszewska**
did not test, am stress, but hopefully it works as intended.
pretty much transformers has a score associated with each positive and negative metric. set the bot to trigger only if it was 75% sure the message was positive and negative.
also added uwu praises due to popular demant

### CODE REVIEW

**Comment by aadibajpai** [comment]
looks nice!

File: bot.py [hunk 1]
```diff
@@ -62,6 +62,28 @@ def get_insult(name):
     return random.choice(insults)
 
 
+#added list of praises
+def get_praise(name):
+    insults = [
```
Comment (aadibajpai) [inline bot.py]: should be praises

File: bot.py [hunk 2]
```diff
@@ -102,9 +124,13 @@ async def on_message(message):
             await message.channel.send(f"Daddy Roth says: don't forget to {think_str}")
         if message.channel.id in target_channel_id and re.search(r"\bbot\b", message.content.lower()):
             result = classifier(message.content.lower())[0]
-            if result["label"] == "NEGATIVE":
+            #add in confidence count for positive and negative comments 
+            if result["label"] == "NEGATIVE" and result["score'] >= 0.75:
```
Comment (aadibajpai) [inline bot.py]:
```suggestion
            if result["label"] == "NEGATIVE" and result["score"] >= 0.75:
```
should use a double quote consistently

File: bot.py [hunk 3]
```diff
@@ -102,9 +124,13 @@ async def on_message(message):
...
                 real_message = get_insult(message.author.display_name)
                 await message.reply(real_message)
+            elif result["label"] == "POSITIVE" and result["score'] >= 0.75:
```
Comment (aadibajpai) [inline bot.py]: same

Comment (KarolinaGroszewska) [reply bot.py]: oop

File: bot.py (hunk 1 above)
Comment (KarolinaGroszewska) [reply bot.py]: but it was insult so i made it praise to match :(

File: bot.py (hunk 2 above)
Comment (KarolinaGroszewska) [reply bot.py]: yes sir

File: bot.py (hunk 1 above)
Comment (aadibajpai) [reply bot.py]: look at the variable bro

Comment (aadibajpai) [reply bot.py]: line 67

Comment (KarolinaGroszewska) [reply bot.py]: o that variable
yes 
hmmphh fine

**Review by aadibajpai — APPROVED** [review]

**Pull request merged**